collections/ansible_collections/demo/cloud/roles/aws/tasks/create_vm.yml

```diff
@@ -36,6 +36,7 @@
     tags:
       blueprint: "{{ aws_blueprint }}"
       purpose: "{{ aws_purpose_tag }}"
+      role: "{{ aws_role_tag | default('demo') }}"
```

Comment:
i don't think the demo.cloud.aws role is used anymore when creating VMs, everything was switched over to to the [aws.infrastructure_config_demos](https://github.com/ansible-content-lab/aws.infrastructure_config_demos) collection a while back.

Reply:
removed